stack/base/Dockerfile

```diff
@@ -16,20 +16,21 @@ WORKDIR /opt/
 
 ARG AIIDA_VERSION
 
+# Pin shared requirements in the base environemnt.
+# The lower bound of aiida-core is set to the version of aiida-core to prevent
+# lower aiida-core DB version installed and trigger database downgrade.
+RUN echo "pip==23.0.1" > /opt/requirements.txt && \
+     echo "aiida-core>=${AIIDA_VERSION},<3" >> /opt/requirements.txt
+
 # Install the shared requirements.
-COPY requirements.txt .
 RUN mamba install --yes \
      aiida-core=${AIIDA_VERSION} \
      mamba-bash-completion \
-     --file requirements.txt \
+     --file /opt/requirements.txt \
      && mamba clean --all -f -y && \
      fix-permissions "${CONDA_DIR}" && \
      fix-permissions "/home/${NB_USER}"
 
-
-# Pin shared requirements in the base environemnt.
-RUN cat requirements.txt | xargs -I{} conda config --system --add pinned_packages {}
```

Comment:
Why was this command removed?

Reply:
Ah, by accident when I switched the lines. I added it back.